README.md

```diff
@@ -33,6 +33,8 @@ Ensure both the forms-admin and forms-runner services are also configured to use
 
 ### Running the tests locally
 
+The tests expect an active group to exist called "End to end tests", which the test user belongs as a group admin. This name can be overridden by setting the environment variable `GROUP_NAME`.
```

Comment:
Is this group and the corresponding membership for the E2E test user created manually through the UI in each env?

Reply:
yes, because it has to be active. It's a bit tricky to create this group before the feature flag is enabled.

Reply:
I'm going to merge this so we can test it but were happy to look at this if you have comments about it.